src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -347,11 +303,10 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
     /**
      * Saves this object.
      *
-     * @param {SaveOptions} saveOptions?
      * @return {Promise}
      * @resolved {String} - The id of the doc
      */
-    this.save = (saveOptions = {}) => {
+    this.save = () => {
```

Comment:
I don't think we can get rid of confirmOverwrite.  In saved object management, we want to confirm overwrite on importing all the objects.  I think with this new implementation it won't warn because it'll use the update call instead of create.

You can see where `confirmOverwrite: true` is passed in in `kibana/src/core_plugins/kibana/public/management/sections/objects/_objects.js`